jaxrs-clients/src/main/java/com/palantir/remoting2/jaxrs/feignimpl/FeignSerializableErrorErrorDecoder.java

```diff
@@ -45,7 +45,6 @@ public Exception decode(String methodKey, Response response) {
         } catch (IOException e) {
             return new RuntimeException("Cannot get input stream from response: " + e.getMessage(), e);
         }
-        return SerializableErrorToExceptionConverter.getException(contentTypes, response.status(), response.reason(),
-                body);
+        return SerializableErrorToExceptionConverter.getException(contentTypes, response.status(), body);
```

Comment:
the reason is null anyway in http/2 and can be statically derived from the status.